### ISSUE TITLE
Reflection support

### ISSUE THREAD

**Issue by ansarizafar** [cjihrig/grpc-server-js#4]
Is their any plan to support reflection

**Comment by cjihrig** [comment]
My understanding from https://github.com/grpc/grpc-node/issues/79 is that reflection doesn't need to be built in, and can be implemented by applications. Is that not the case?

**Comment by ansarizafar** [comment]
Yes I have seen that issue. I am new to GRPC and I am not sure how to implement it but GRPC implementations in other languages have built-n support for reflection. There is another interesting GRPC project for nodejs where we are discussing reflection implementation https://github.com/malijs/mali/issues/50
